src/main/java/com/example/demo/client/JsonPlaceholderWebClient.java

```diff
@@ -0,0 +1,135 @@
+package com.example.demo.client;
+
+import com.example.demo.enumeration.MethodTypeEnum;
+import com.example.demo.projectVeriables.ProjectVeriables;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.ClientResponse;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.logging.Logger;
+
+@Component
+public class JsonPlaceholderWebClient {
+    private static final Logger logger = Logger.getLogger(JsonPlaceholderWebClient.class.getName());
+
+    @Autowired
+    WebClient client;
+
+    public ClientResponse sendGet(String url, MethodTypeEnum query, HttpHeaders headers) {
```

Comment:
let's move sendGet, sendPost, sendDelete, sendPatch, sendPut to the bottom of this class - will be more readable